addon-test-support/pages/-private/base-pop-menu.js

```diff
@@ -51,7 +51,11 @@ export default PageObject.extend({
     /**
      * Returns whether or not the trigger has been marked as active
      */
-    isActive: hasClass('is-active')
+    isActive: hasClass('is-active'),
+
+    hasAriaPopup: attribute('aria-haspopup'),
```

Comment:
Do you think that we'll need to test these directly in Iverson? I'd think that testing them in `ice-pop` with unit tests would be enough, but I'm not 💯  on that

Reply:
I wouldnt test them in iverson, seems overkill to me

Reply:
Ok, I'd move these to the tests themselves then

Reply:
Oh I see why you asked, I thought the point of putting this here was so I didn't have to repeat it for all the others. I'm confused now lol

Reply:
Ah, that's a good point I hadn't considered. I think it's fine then, it helps us dry up tests here and most likely people won't rely on it in Iverson too much